forwards_paper.tex

```diff
@@ -64,25 +99,38 @@
 
 
 \begin{abstract}
-    In this paper we describe how to
-    efficiently record the entire genetic history of a population
-    during a forwards-time, individual-based population genetics simulation.
-    This dramatically reduces the computational burden of tracking individual genomes
-    by simulating only those loci that may affect reproduction (those having non-neutral variants),
-    and recording history as a \emph{succinct tree sequence} as introduced in the software package \msprime,
+    In this paper we describe how to efficiently record the entire genetic history of a population
+    in forwards-time, individual-based population genetics simulations with arbitrary breeding models,
+    population structure and demography.
+    This approach dramatically reduces the computational burden of tracking individual genomes
+    by allowing us to simulate only those loci that may affect reproduction
+    (those having non-neutral variants).
+    The genetic history of the population is recorded as a succinct tree sequence
+    as introduced in the software package \msprime,
     on which neutral mutations can be quickly placed afterwards.
-    We implement the method in two popular forwards-time simulation frameworks,
-    which speeds up large, whole-genome simulations by one to two orders of magnitude.
-    In addition to speed, the method has several advantages:
+    Recording the results of each breeding event
+    results in a linear growth in the amount of information that must be stored.
+    However, there is a great deal of redundancy in this information,
+    since the majority of individuals simulated over the course of the simulation
+    will leave no descendents in the eventual sample.
```

Comment:
s/will leave no descendents in the eventual sample./will leave no descendents in the final population./

Reply:
ah: this is actually wrong.  Usually I think something like 60% of the individuals leave descendants; and in any case that's not where most of the redudancy comes from. I'll reword this.

Reply:
Huh, not sure how to word this. Like "the majority of individuals aren't relevant to the genealogy of the final generation." or "coalescent events don't occur in most individuals"... I've put just 
>     Recording the results of each breeding event requires storage that grows linearly with time,
>    but there is a great deal of redundancy in this information.

Reply:
Ah nice point. The new words look good.